RELEASING.md

```diff
@@ -13,7 +13,7 @@ and assign it to @bencochran or @aquageek.*
 ---
 1. Merge an update of [the change log](CHANGELOG.md) with the changes since the last release.
 
-1. Make sure you're on the `master` branch (or fix branch, e.g. `v0.1-fixes`).
+1. Create a new release branch `release-v0.1.x` (or checkout fix branch, e.g. `release-v0.0.x`).
```

Comment:
This isn't strictly required, but probably a good habit to get into anyway. Thanks for adding.